<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>870</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>870</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>891</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Information</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>871</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="full_label">
           <property name="text">
            <string>Full:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="full_output">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="empty_label">
           <property name="text">
            <string>Empty :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="empty_output">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="date_lable">
           <property name="text">
            <string>Date :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="data_output">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="time_lable">
           <property name="text">
            <string>Time :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time_output">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>730</y>
     <width>171</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="start_buttom">
      <property name="text">
       <string>Start </string>
      </property>
      <property name="icon">
       <iconset resource="img/icons.qrc">
        <normaloff>:/newPrefix/play_arrow_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/newPrefix/play_arrow_FILL0_wght400_GRAD0_opsz48.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="end_buttom">
      <property name="text">
       <string>End </string>
      </property>
      <property name="icon">
       <iconset resource="img/icons.qrc">
        <normaloff>:/newPrefix/stop_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/newPrefix/stop_FILL0_wght400_GRAD0_opsz48.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="activate_buttom">
      <property name="text">
       <string>Activate </string>
      </property>
      <property name="icon">
       <iconset resource="img/icons.qrc">
        <normaloff>:/newPrefix/motion_sensor_active_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/newPrefix/motion_sensor_active_FILL0_wght400_GRAD0_opsz48.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_buttom">
      <property name="text">
       <string>Save </string>
      </property>
      <property name="icon">
       <iconset resource="img/icons.qrc">
        <normaloff>:/newPrefix/download_FILL0_wght400_GRAD0_opsz48.png</normaloff>:/newPrefix/download_FILL0_wght400_GRAD0_opsz48.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>720</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/newPrefix/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>logo_label</zorder>
  <zorder>img_label</zorder>
  <zorder>groupBox</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
 </widget>
 <resources>
  <include location="img/icons.qrc"/>
 </resources>
 <connections/>
</ui>
